WordPress/Classes/ViewRelated/Jetpack/Jetpack Scan/JetpackScanCoordinator.swift

```diff
@@ -205,6 +211,17 @@ class JetpackScanCoordinator {
         supportVC.showFromTabBar()
     }
 
+    public func openJetpackSettings() {
+        guard let siteID = blog.dotComID as? Int,
+              let jetpackSettingsURL = AppConstants.jetpackSettingsURL(siteID: siteID) else {
+            view.presentNotice(with: Strings.jetpackSettingsNotice.title, message: nil)
+            return
+        }
+
+        let controller = WebViewControllerFactory.controller(url: jetpackSettingsURL)
+        view.showJetpackSettings(webViewController: controller)
+    }
+
```

Comment:
What do you think about including all this logic in the `showJetpackSettings` method? Then it can be reduced down to

```suggestion
    public func openJetpackSettings() {
        view.showJetpackSettings(with: blog.dotComID)
    }

```

Reply:
Fixed! 699aa24